docker-compose.yml

```diff
@@ -51,9 +57,76 @@ services:
       WAIT_HOSTS: redis:6379, postgres:5432, camunda:8080
       WAIT_TIMEOUT: 300
       WAIT_SLEEP_INTERVAL: 5
+      PLUGIN_REGISTRY_URL: "http://registry:8080"
+  backend:
```

Comment:
I am not sure if we should include UI and backend here. We also may want to use profiles to only start the additional components when actually required.